mcs/class/System.Data/referencesource.sources

```diff
@@ -18,19 +18,12 @@
 ../referencesource/System.Data/System/Data/Common/ActivityCorrelator.cs
```

Comment:
Doesn't need to be part of this PR, but I think it'd be good to extract the files which are duplicated between referencesource.sources and mobile_referencesource.sources into a common.sources file.